docs/make.jl

```diff
@@ -58,6 +58,7 @@ makedocs(;
             "Asia Network" => "generated/asia-network/main.md",
             "Hard-core Lattice Gas" => "generated/hard-core-lattice-gas/main.md",
            ],
+        "Performance evaluation" => "performance-evaluation.md",
         "Performance tips" => "generated/performance.md",
```

Comment:
Please also change the last line of performance tips. It mentioned the benchmark but with no link.